R/centrality.R

```diff
@@ -19,20 +19,28 @@
 #
 ###################################################################
 
-#' @rdname betweenness
+#' Deprecated version of `betweenness()`
+#'
+#' @description Use [`betweenness()`] with the `cutoff` argument instead.
+#' `r lifecycle::badge("deprecated")`
 #' @param vids The vertices for which the vertex betweenness estimation will be
 #'   calculated.
-#' @param cutoff The maximum path length to consider when calculating the
-#'   betweenness. If zero or negative then there is no such limit.
-#' @family centrality
+#' @inheritParams betweenness
 #' @export
-estimate_betweenness <- function(graph, vids = V(graph), directed = TRUE, cutoff, weights = NULL, nobigint = TRUE) {
-  if (!missing(nobigint)) {
-    warning("'nobigint' is deprecated since igraph 1.3 and will be removed in igraph 1.4")
-  }
+estimate_betweenness <- function(graph, vids = V(graph), directed = TRUE, cutoff, weights = NULL) {
+
+  lifecycle::deprecate_soft(
+    "1.5.0",
```

Comment:
```suggestion
    "1.6.0",
```

We may need to change to 2.0.0 later.